master/master.cfg

```diff
@@ -841,6 +841,10 @@ def add_env_setup_step(factory, builder_type, enable_ccache=False):
                                   Transform(os.path.join, hexagon_remote_bin, 'host'),
                                   Interpolate('%(prop:HL_HEXAGON_TOOLS)s/lib/iss')]
 
+    # Force Vulkan validation layer on to catch any driver related errors
+    if builder_type.os == 'linux' and builder_type.has_nvidia() and builder_type.handles_vulkan():
+        env['VK_INSTANCE_LAYERS'] = "VK_LAYER_KHRONOS_validation"
+
```

Comment:
Maybe add a comment explaining when Vulkan runs but this environment variable is not set? I don't understand how this condition is different from unconditionally setting this variable. Maybe it doesn't take effect on Windows?

Reply:
The conditional limits it to the builder type that currently enables the vulkan target ... but we could set it unconditionally.  It'll only have an effect when the vulkan target is being used.

Reply:
Added more comments ... and relaxed conditional to only check for `has_vidia()` and `handles_vulkan()`.